### ISSUE TITLE
Editing the name of a new root node that you added attributes loses the attributes

### ISSUE THREAD

**Issue by lovettchris** [microsoft/XmlNotepad#343]
Start with:
```xml
<?xml version="1.0" encoding="utf-8"?>
<root xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://schemas.microsoft.com/vs/2009/dgml file:///F:/OneDriveWork/OneDrive%20-%20Microsoft/Progression/SDK_1/DGML.xsd" xmlns="http://schemas.microsoft.com/vs/2009/dgml" />
```

Load it and rename "root" to "DirectedGraph" and you get this error:

![image](https://github.com/microsoft/XmlNotepad/assets/18707114/386651b0-f05a-4fbc-9e5e-b63b288aae9a)

Save the file and the new file has no attributes...

**Comment by lovettchris** [comment]
Will be fixed in version 2.9.0.7

**Comment by lovettchris** [comment]
Fixed
